src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Sql</PackageId>
     <Description>Azure SQL Management SDK library</Description>
     <AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-    <VersionPrefix>1.8.0-preview</VersionPrefix>
+    <VersionPrefix>1.9.0-preview</VersionPrefix>
     <PackageTags>Microsoft Azure SQL Management;SQL;SQL Management</PackageTags>
   </PropertyGroup>
   <PropertyGroup>
```

Comment:
please add a release note in `<PackageReleaseNotes>` below, e.g. 'Added support for server dns aliases'.